### PR TITLE
Bugfix bgp af delete and update docs for bgp_global and bgp_af

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME

bgp_address_family, bgp_neighbor_address_family

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/iosxr_bgp_address_family.py [hunk 1]
```diff
@@ -478,6 +478,434 @@
...
+# -------------
+# RP/0/0/CPU0:iosxr-02#show running-config router bgp
+# Sat Feb 20 03:49:43.618 UTC
+# router bgp 65536
+#  bgp router-id 192.0.2.1
+#  address-family vpnv4 unicast
+#  vrf vrf1
+#   rd auto
+- name: Merge the provided configuration with the exisiting running configuration
+  cisco.iosxr.iosxr_bgp_neighbor_address_family:
```
Comment (NilashishC) [inline plugins/modules/iosxr_bgp_address_family.py:492]:
```suggestion
  cisco.iosxr.iosxr_bgp_address_family:
```

File: tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 - debug:
-    msg: START iosxr_bgp_address_family gathered integration tests on connection={{ ansible_connection
+    msg: START iosxr_bgp_neighbor_address_family gathered integration tests on connection={{ ansible_connection
```
Comment (NilashishC) [inline tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml:3]:
```suggestion
    msg: START iosxr_bgp_address_family gathered integration tests on connection={{ ansible_connection
```

**Review by NilashishC — CHANGES_REQUESTED** [review]

**Comment by NilashishC** [comment]
Just a few minor docs comments, rest LGTM!

File: tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml (hunk 2 above)
Comment (ashwini-mhatre) [reply tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml:3]: this is from iosxr_bgp_neighbor_address_family module.this was failing on stable 2.9.so fixed that

File: plugins/modules/iosxr_bgp_address_family.py (hunk 1 above)
Comment (ashwini-mhatre) [reply plugins/modules/iosxr_bgp_address_family.py]: ACK,made changes

File: tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml (hunk 2 above)
Comment (NilashishC) [reply tests/integration/targets/iosxr_bgp_neighbor_address_family/tests/common/gathered.yaml:3]: Oh got it.

**Review by NilashishC — APPROVED** [review]

**Pull request merged**